brain/morphology.h

```diff
@@ -0,0 +1,122 @@
+
+/* Copyright (c) 2013-2015, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef BRAIN_MORPHOLOGY
+#define BRAIN_MORPHOLOGY
+
+#include <brain/api.h>
+#include <brain/types.h>
+
+#include <boost/noncopyable.hpp>
+
+namespace brain
+{
+
+/**
+ *  A higher level interface to morphological data than the raw morphology.
+ *
+ * @version usntable
+ */
+class Morphology : public boost::noncopyable
+{
+public:
+    /**
+     * Create the morphology and load all the data.
+     */
+    BRAIN_API explicit Morphology(
+        const std::string& source,
+        brion::MorphologyRepairStage stage = brion::MORPHOLOGY_UNDEFINED );
+    /**
+     * Create the morphology and load all the data.
+     */
+    BRAIN_API explicit Morphology(
+        const brion::Morphology& morphology,
```

Comment:
I have a reason for keeping at least the constructor that takes a MorphologyPtr, which is to be able to reuse the output of brion::morphologies::load.  I'm not using the constructor that takes a const brion::Morphology&, so it could be removed. I added simply for convenience.